lib/job/arTestJob.class.php

```diff
@@ -27,14 +27,15 @@
 
 class arTestJob extends arBaseJob
 {
-  public function run($parameters)
+  // Required parameters:
+  // - Parameters that are mandatory for the job
+  // - They are checked at the begining of the job from arBaseJob run() function
+  // - 'id' and 'name' are required parameters for all jobs and they are added in arBaseJob
+  // - The job will fail if any of the required paramaters are missing
+  protected $requiredParameters = array();
```

Comment:
Looking at checkRequiredParameters() it seems that removing this is ok as isset() is used, right?